pkg/flowaggregator/flowaggregator.go

```diff
@@ -403,12 +380,16 @@ func (fa *flowAggregator) flowExportLoop(stopCh <-chan struct{}) {
 
 func (fa *flowAggregator) sendFlowKeyRecord(key ipfixintermediate.FlowKey, record *ipfixintermediate.AggregationFlowRecord) error {
 	isRecordIPv4 := fa.aggregationProcess.IsAggregatedRecordIPv4(*record)
+	startTime, err := fa.getRecordStartTime(record.Record)
+	if err != nil {
+		klog.ErrorS(err, "Cannot find record start time")
+	}
 	if !fa.aggregationProcess.AreCorrelatedFieldsFilled(*record) {
-		fa.fillK8sMetadata(key, record.Record)
+		fa.fillK8sMetadata(key, record.Record, *startTime)
```

Comment:
in case of an error, `getRecordStartTime` will return a `startTime` value equal to `nil`. While you log the error, you continue with normal function execution and you later dereference `startTime` with `*startTime`. This would case a segfault. You have several options in case of error, and maybe you can choose the most appropriate:
1) use `time.Now()` as `startTime`
2) skip filling the metadata that requires `startTime`
3) interrupt the function execution

Reply:
Fixed. It should return directly. Thanks.